### DISCUSSION TITLE
Read Ordering Only

### DISCUSSION THREAD

**Discussion by saffie91** [mindee/doctr#1539] [Q&A]
### 🚀 The feature

Is there a way to use the read ordering of this model only. Given another OCR's input? If not would it be possible?

Also how do you do the read ordering? Is it part of the CRNN or is it a function somewhere?

### Motivation, pitch

I'm working on having something similar work on a wide variety of languages and non imported documents. Would like to be able to update my method with better read ordering.

**Comment by felixdittrich92** [comment]
Hey @saffie91 😊
Ordering isn't model specific everything is done in https://github.com/mindee/doctr/blob/0d849152a852cb55b3fb0cc0e5e602600349d97d/doctr/models/builder.py#L19
